pkg/Library/Data/Nodes/GridNode.js

```diff
@@ -3,9 +3,11 @@
  * Copyright 2023 NeonFlan LLC
  * SPDX-License-Identifier: BSD-3-Clause
  */
+// [DEPRECATED] This node should be deprecated and replaced by TableNode.
+// TODO(maria): Fix the public graphs and delete it!
```

Comment:
i promise to remove this within the next day or two, i just want the older graphs to work while i'm updating them.

Reply:
No hurry